components/Modals/UpdateGameMetadataModal.tsx

```diff
@@ -346,7 +420,31 @@ export const UpdateGameMetadataModal: React.FC = () => {
           )}
           {showError && noChanges && (
             <FormHelperText color="red">
-              New name, description, or emblem must be different from the old
+              New name, description, emblem, or base token URI must be different
+              from the old
+            </FormHelperText>
+          )}
+        </FormControl>
+        <FormControl isInvalid={showError && (!newBaseTokenURI || noChanges)}>
+          <Flex align="center">
+            <FormLabel>Base Character Token URI</FormLabel>
+            <Tooltip label="This base URI will be used at the start of every character token metadata URI.">
+              <Image
+                alt="down arrow"
+                height="14px"
+                mb={2}
+                src="/icons/question-mark.svg"
+                width="14px"
+              />
+            </Tooltip>
+          </Flex>
+          <Input
+            onChange={e => setNewBaseTokenURI(e.target.value)}
+            value={newBaseTokenURI}
+          />
+          {showError && !newBaseTokenURI && (
+            <FormHelperText color="red">
+              A base token URI is required
```

Comment:
Flow looks and works well but I wonder if we need to allow users to set the base token uri. If they make a mistake of updating it then it will be a disaster.. What do you think @ECWireless ? 

I was thinking we do the base token uri update similar to the character metadata update that you're doing. A fixed uri that will be sent in the background..

Reply:
Removed ability to update the base token URI in the frontend: e4db133ce344c2ff0a051780d432e17708d779eb